app/controllers/attendances_controller.rb

```diff
@@ -1,4 +1,40 @@
 class AttendancesController < ApplicationController
   def index
   end
+
+  def new
+    @attendance = Attendance.new
+    @company = Company.where(disable: false)
```

Comment:
What about adding a scope to `Company` instead?

```ruby
# company.rb
class Company < ApplicationRecord
  # ...

  scope :enabled, -> { where(disabled: false) }
end 
```

After that, you can use it like this:

```ruby
@company = Company.enabled
```

https://guides.rubyonrails.org/active_record_querying.html#scopes